packages/strapi-design-system/src/CarouselInput/CarouselInput.js

```diff
@@ -25,13 +25,9 @@ export const CarouselInput = ({
   const generatedId = useId('carouselinput', id);
 
   return (
-    <Field hint={hint} error={error} id={generatedId}>
+    <Field hint={hint} error={error} id={generatedId} required={required}>
       <Stack spacing={1}>
-        {label && (
-          <FieldLabel required={required} action={labelAction}>
-            {label}
-          </FieldLabel>
-        )}
+        {label && <FieldLabel action={labelAction}>{label}</FieldLabel>}
```

Comment:
@ronronscelestes Is this label missing an id?

Reply:
it gets it from `Field` 👀 @gu-stav 
<img width="476" alt="image" src="https://user-images.githubusercontent.com/71838159/206168144-54892bc7-b32a-4cd5-b18b-1e420d536dc9.png">
<img width="596" alt="image" src="https://user-images.githubusercontent.com/71838159/206168238-9f3725af-3b9f-484d-892b-b4ff9bacc546.png">

Reply:
Yes, that's why we don't have to pass it on `<FieldLabel />`. There are a few where we pass it: `<FieldLabel id="..." />`

Reply:
ouh will check that thanks!

Reply:
done [here](https://github.com/strapi/design-system/pull/767/commits/a847144d9e83aedd1fd7d9a9acf9d9c634a2ff63) @gu-stav

Reply:
Nice, thanks! Added a follow-up bug here: https://github.com/strapi/design-system/issues/800